packages/form-fields/text/style.module.css

```diff
@@ -0,0 +1,25 @@
+.root {
+  composes: wpl-variables input-root from '../style.module.css';
```

Comment:
Fixed this specific instance, but I need to sync up with design because, at least by my understanding, there's no concept of a "focused error" state since both focus and error states use the same 1px border, making it impossible to overlap them.